src/chunkserver/network_worker_thread.h

```diff
@@ -168,14 +168,206 @@ struct ChunkserverEntry {
+
+	/// Attempts to re-establish a connection to the next chunkserver.
+	/// Implements a retry mechanism to ensure that the connection
+	/// is eventually established
+	void retryConnect();
+
+	/// Checks and processes the next packet in the input buffer.
+	void checkNextPacket();
+
+	/// Processes a received packet based on its type.
+	///
+	/// @param type The type of the packet.
+	/// @param data Pointer to the packet data.
+	/// @param length The length of the packet data.
+	void gotPacket(uint32_t type, const uint8_t *data, uint32_t length);
+
+	/* IDLE Operations */
+
+	/// Answers to a ping message with the given data and length.
+	void ping(const uint8_t *data, PacketHeader::Length length);
+
+	/// Initializes a read operation
+	///
+	/// @param data Pointer to the buffer containing the information to read.
+	/// @param type The type of the packet.
+	/// @param length The length of the packet data.
+	void readInit(const uint8_t *data, PacketHeader::Type type,
+	              PacketHeader::Length length);
+
+	/// Continues a previously started read operation.
+	///
+	/// Processes the remaining data to be read from the chunkserver. If all
+	/// data has been read, it sends a read status message and closes the chunk.
+	/// Otherwise, it prepares the next part of the read operation.
+	///
+	/// @see ChunkserverEntry::readInit
+	void readContinue();
+
+	/// Requests a data prefetch operation.
+	/// Prefetch in this context means reading data from the disk and storing it
+	/// in the page cache.
+	void prefetch(const uint8_t *data, PacketHeader::Type type,
+	              PacketHeader::Length length);
+
+	/// Callback for when a read operation finishes.
+	static void readFinishedCallback(uint8_t status, void *entry);
+	/// Callback after delayed close operations.
+	static void delayedCloseCallback(uint8_t status, void *entry);
+	/// Callback for when a write operation finishes.
+	static void writeFinishedCallback(uint8_t status, void *entry);
+	/// Callback for legacy chunk block retrieval completion.
+	static void sauGetChunkBlocksFinishedLegacyCallback(uint8_t status,
+	                                                    void *entry);
+	/// Callback for chunk block retrieval completion.
+	static void sauGetChunkBlocksFinishedCallback(uint8_t status, void *entry);
+	/// Callback for chunk block retrieval completion.
+	static void getChunkBlocksFinishedCallback(uint8_t status, void *entry);
+
+	/// Retrieves chunk blocks from the given information.
+	void getChunkBlocks(const uint8_t *data, uint32_t length);
+
+	/// Retrieves chunk blocks from the given information using the new way.
+	void sauGetChunkBlocks(const uint8_t *data, uint32_t length);
+
+	/// Retrieves the list with the HDDs information.
+	void hddListV2(const uint8_t *data, uint32_t length);
+
+	/// Lists the disk groups (if the DiskManager supports it).
+	void listDiskGroups([[maybe_unused]] const uint8_t *data,
+	                    [[maybe_unused]] uint32_t length);
```

Comment:
Consider using the same rule for `unused` parameter. Either `maybe_unused` or `(void) var;` inside the function...